TaskModuleUIConstants.js does not exist

## Sample information
1. Sample type: /opt/BotBuilder-Samples/samples/javascript_nodejs/54.teams-task-module
2. Sample language: nodejs
3. Sample name: 54.teams-task-module

## Describe the bug
Error: Cannot find module '../models/TaskModuleUIConstants'

## To Reproduce
npm start

## Expected behavior
Start project

##
OS Ubuntu 18.04

##
SDK 4

##
Branch default

## Solution
cd models
cp taskmoduleuiconstants.js TaskModuleUIConstants.js

--- a/samples/javascript_nodejs/54.teams-task-module/bots/teamsTaskModuleBot.js
+++ b/samples/javascript_nodejs/54.teams-task-module/bots/teamsTaskModuleBot.js
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 const { TeamsActivityHandler, MessageFactory, CardFactory } = require('botbuilder');
-const { TaskModuleUIConstants } = require('../models/TaskModuleUIConstants');
+const { TaskModuleUIConstants } = require('../models/taskModuleUIConstants');
 const { TaskModuleIds } = require('../models/taskmoduleids');
 const { TaskModuleResponseFactory } = require('../models/taskmoduleresponsefactory');
 

